src/components/Input/type.tsx

```diff
@@ -0,0 +1,29 @@
+export enum InputType {
+  TEXT = 'text',
+  PASSWORD = 'password',
+  EMAIL = 'email',
+  NUMBER = 'number',
+  DATE = 'date',
+  CHECKBOX = 'checkbox',
+  RADIO = 'radio',
+  FILE = 'file',
+  SUBMIT = 'submit',
+  RESET = 'reset',
+  BUTTON = 'button',
+  SELECT = 'select'
+}
+
+export type SelectOption = {
+  value: string;
```

Comment:
it should also have `label` in case if it should be shown differently in the UI

Reply:
Label added to each input